_commands/basics/comm.md

```diff
@@ -0,0 +1,60 @@
+---
+---
+
+comm
+---
+Compares two sorted files line by line and writes three columns to standard output.
+<!-- one line explanation would go here -->
+
+<!-- minimal example -->
+~~~ bash
+# content of file1.txt
+$ cat file1.txt
```

Comment:
Show the contents of file1.txt and file2.txt next to each other in the same lines. That way they don't take up as much space in the example.